lib/index.js

```diff
@@ -1,55 +1,65 @@
 var fs    = require('fs');
 var Joi   = require('joi');
+var glob  = require('glob');
 var path  = require('path');
 
+var internals = {};
```

Comment:
Since the internal functions have been removed, I think the `internals` object isn't necessary anymore.